docs/02-tutorials/01-intro/07-storage-set-up.mdx

```diff
@@ -4,16 +4,239 @@ title: 6. Set-up Storage
 tags: [tutorial]
 ---
 
-> :construction_worker: writing in progress
+The blindnet devkit storage can be handled via two components:
 
-A [Data Access Component](/docs/storage/data-access) is here to transform requests from the [Privacy Computation Engine](/docs/computation) to efficiently execute automated operations on your data storage.
+1. a [Data Access Component](/docs/storage/data-access) (DAC)
+2. a storage service, with optionnal help from a devkit storage connector library (currently, only Java is supported, via the  [Java Storage Connector library](https://github.com/blindnet-io/storage-connector-java))
 
-You can choose to use the [Capture Component](/docs/data-capture) and directly store the captured data in blindnet's preconfigured storage, or your own storage service.
+You can choose to use the [Capture Component](/docs/data-capture) and directly store the captured data in blindnet's preconfigured storage, or your own storage service. You can also directly connect the Data Access Component to your own storage service without using the Capture Component. For see sake of simplicity, we'll choose this option for this tutorial.
 
-You can also directly connect the Data Access Component to your own storage service without using the Capture Component. For see sake of simplicity, we'll choose this option for this tutorial.
+For that, we'll set up a dedicated demo storage service in Java, and connect it to the Capture Component using the.
 
-For that, we'll set up a dedicated demo storage service in Java, and connect it to the Capture Component using the [Java Storage Connector library](https://github.com/blindnet-io/storage-connector-java).
+## Pre-requisites
+
+The following steps require **git**, a **Java Development Kit**, **sbt**, and **docker** to be installed on your machine.
+
+Refer to the [associated section](http://localhost:3000/docs/tutorials/intro/PCE-set-up#pre-requisites) of the first step of this tutorial for more information.
```

Comment:
I guess this localhost link should be fixed

Reply:
indeed

```suggestion
Refer to the [associated section](https://blindnet.dev/docs/tutorials/intro/PCE-set-up#pre-requisites) of the first step of this tutorial for more information.
```